### DISCUSSION TITLE
Get the file instead of download

### DISCUSSION THREAD

**Discussion by dittuxdev** [stephanrauh/ngx-extended-pdf-viewer#1649] [General]
Hi everyone, there is any way to get the file in base64, file, or bytres or anything, instead of download the file when we click download button?

**Comment by stephanrauh** [comment]
Yes, probably. Have a look at this: https://github.com/stephanrauh/ngx-extended-pdf-viewer/issues/1592#issuecomment-1331362058

That's only a sketch. I plan to add the programatic download as a regular feature with version 16.0.0.
